promgen/templates/promgen/global_messages.html

```diff
@@ -0,0 +1,8 @@
+{% for message in messages %}
+<div class="alert {% if message.tags %}alert-{{ message.tags }}{% endif %}" role="alert">
```

Comment:
ref: https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax
This is a little difficult to understand, so how about this ?

```suggestion
<div class="alert" :class="{ alertMessageTags }" role="alert">
```

and you can write such a property.

```js
computed: {
  alertMessageTags: function () {
    return {
      `alert-${message.tags}`: this.message.tags
    }
  }
}
```

Reply:
Ah, that part is actually still django template code it's unfortunate that the vue js and django templates look the same (so in some places I had to have django ignore vuejs blocks)

Reply:
ok I understand.